packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -49,6 +49,8 @@ export class GraphQLRequestClient {
         query,
         variables,
       });
+      console.log(query);
+      console.log(JSON.stringify(variables, null, 2));
```

Comment:
console.logs removed (thanks!)
The pretty print of DocumentNode no longer applies since we are not actually converting our queries to ASTs (we're just using prettier for syntax highlighting)